Core/src/org/sleuthkit/autopsy/casemodule/Case.java

```diff
@@ -1798,16 +1810,19 @@ private void openCaseData(ProgressIndicator progressIndicator) throws CaseAction
             } else if (UserPreferences.getIsMultiUserModeEnabled()) {
                 try {
                     caseDb = SleuthkitCase.openCase(databaseName, UserPreferences.getDatabaseConnectionInfo(), metadata.getCaseDirectory());
-
                 } catch (UserPreferencesException ex) {
-                    throw new CaseActionException(NbBundle.getMessage(Case.class,
-                            "Case.databaseConnectionInfo.error.msg"), ex);
-
+                    throw new CaseActionException(Case_databaseConnectionInfo_error_msg(), ex);
                 }
             } else {
-                throw new CaseActionException(NbBundle.getMessage(Case.class,
-                        "Case.open.exception.multiUserCaseNotEnabled"));
+                throw new CaseActionException(Case_open_exception_multiUserCaseNotEnabled());
             }
+        } catch (TskUnsupportedSchemaVersionException ex) {
+            throw new CaseActionException(
+                    Bundle.Case_unsupportedSchemaVersionMessage(
+                            ex.getUnsupportedVersion().toString(),
+                            Version.getName(),
+                            ex.getCurrentVersion().getMajor()),
+                    ex);
```

Comment:
Please see the comment in the corresponding sleuthkit pull request regarding the design of the TskUnsupportedSchemaVersion class, as it would affect the construction here of the displayable "user-friendly" message to satisfy the current convention of the usage of CaseActionException. Note that as soon as time permits, I want to do away with that convention. We would be much better served by a hierarchy of CaseActionException subtypes that allow clients to assemble meaningful error messages based on exception type, and could make such a change without breaking backwards compatibility of the Case public API.